main.cpp

```diff
@@ -8,17 +8,23 @@
 int
 main() {
 
-	auto conn = sdbus::createSessionBusConnection();
+	std::unique_ptr<sdbus::IConnection> conn;
+	try {
+		conn = sdbus::createSessionBusConnection();
+	} catch (std::runtime_error &e) {
+		std::cerr << "There was an error creating DBus connection." << std::endl;
+		std::cerr << "Error: " << e.what() << std::endl;
```

Comment:
Should the program not try exiting here?

Reply:
It should definitely exit here, my bad.